e2e_testing/torchscript/basic.py

```diff
@@ -1179,3 +1179,21 @@ def forward(self, a):
 @register_test_case(module_factory=lambda: BoolTensorReturnMixedModule())
 def BoolTensorReturnMixedModule_basic(module, tu: TestUtils):
     module.forward(torch.tensor([[1, 0], [0,1]], dtype=torch.bool))
+
+# ==============================================================================
+class TModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([3, 4], torch.float32, True),
```

Comment:
Check for dynamic dimension. [-1, -1].

Reply:
Is it okay to do all three rank classes in dynamic or do I need 6 classes 3 for static and three for dynamic?

Reply:
You can do all dynamic